ios/RPCModule.m

```diff
@@ -276,6 +278,7 @@ -(NSString*) createNewWallet:(NSString*)server
                  birthday:(NSString*)birthday
                  server:(NSString*)server
                  chainhint:(NSString*)chainhint
+                 monitorMempool:(NSString*)monitorMempool
```

Comment:
You need to follow the same pattern that you used in `android`, remove this line & put `true` directly in the call to initfromufvk, like this: https://github.com/Oscar-Pepper/zingo-mobile/blob/019fe6590055abb8286272d55f6fa873fe6cfeb2/android/app/src/main/java/org/ZingoLabs/Zingo/RPCModule.kt#L93